smartmirror.py

```diff
@@ -10,8 +10,7 @@
 import feedparser
 from PIL import Image, ImageTk
 
-ip = '<IP>'
-country_code = 'us'
+country_code = None  # set this here if you don't want news from your current location
 weather_api_token = '<TOKEN>'
```

Comment:
I'll try and set aside some time to go through the code base and see if there's anything I can nit pick 😉. I probably won't be adding any new features unless they're built on top of #5, which in my opinion is a huge win.